docs/schema.md

```diff
@@ -0,0 +1,9 @@
+# Portal Data Schema
+
+What are datasets, runs, and annotations in the Data Portal?
+
+1. A dataset is a community contributed set of image files for tilt series, reconstructed tomograms, and if available, cellular and/or subcellular annotation files. Every dataset contains only one sample type prepared and imaged with the same conditions. The dataset title, such as `S. pombe cryo-FIB lamellae acquired with defocus-only`, summarizes these conditions. Samples can be a cell, tissue or organism; intact organelle; in-vitro mixture of macromolecules or their complex; or in-silico synthetic data, where the experimental conditions are kept constant. Downloading a dataset downloads all files, including all avilable tilt series, tomograms, and annotations.
+2. A run is one experiment, or replicate, associated with a dataset, where all runs in a dataset are of the sample type, sample preparation, and imaging configuration. Typically a run produces only one tilt series. However, for a single tilt series, there can be multiple tomograms, each corresponding to the specific algorithm used to process the raw data. When downloading a run, you may download only the tomograms or the tomograms and available annotations.
+3. An annotation is a point or segmentation indicating the location of a macromolecular complex in the tomogram. All tomograms in the Data Portal have point annotations of (at least one) macromolecular complex, such an a ribosome. On the run page, you may choose to download tomograms with their annotations.
```

Comment:
> such an a ribosome

minor spelling: such as a ribosome

I don't think all tomograms in the data portal will have point annotations. i think there will be some without any annotations. would be good to double confirm with Anchi

Reply:
It's true that there will be datasets that won't have point annotations supplied by the data depositors. E.g. dataset 10010 will only have semantic segmentation masks for now.

Reply:
Thank you for clarifying @uermel! I made a PR to fix that statement on the landing page [here.](https://github.com/chanzuckerberg/cryoet-data-portal/pull/164)